assets/js/components/HostDetails/HostDetails.jsx

```diff
@@ -24,6 +25,16 @@ import {
   sapInstancesTableConfiguration,
 } from './tableConfigs';
 
+const semver = require('semver');
+
+export const agentVersionWarnings = (agentVersion) => {
+  if (semver.lt(agentVersion, '2.0.0')) {
+    return 'Agent version 2.0.0 or greater is required for new checks engine.';
```

Comment:
I'd phrase it as: "...required to use the new checks engine." or "...required for the new checks engine"

Reply:
This was coming directly from @abravosuse 
What should we put?

Reply:
I agree,  "for **the** new checks engine", the article is missing.

Reply:
Corrected to `required for the...`

Reply:
"Agent version 2.0.0 or greater is required for the new checks engine." is fine with me.